includes/blocks.inc

```diff
@@ -694,6 +694,10 @@ function islandora_solr_advanced_search_form_submit($form, &$form_state) {
     $field = islandora_solr_lesser_escape($field);
     // Add query.
     if (!empty($search)) {
+      if (variable_get('islandora_solr_advanced_search_block_lucene_syntax_escape', TRUE)) {
+        $escape_charset = variable_get('islandora_solr_advanced_search_block_lucene_charset_default', "+,-,&&,||,!,(,),{,},[,],^,~,:");
+        $search = islandora_solr_facet_query_escape($search, explode(',', $escape_charset));
```

Comment:
Why not just use [islandora_solr_lesser_escape](https://github.com/Islandora/islandora_solr_search/blob/7.x/includes/utilities.inc#L139) or [islandora_solr_facet_escape](https://github.com/Islandora/islandora_solr_search/blob/7.x/includes/utilities.inc#L165).

Either of these would resolve the issue in the JIRA ticket and then there is no chance a user could remove a needed character. Or is there a use case for an expanded filter that I missed?